variables.tf

```diff
@@ -30,8 +30,8 @@ variable "webhook_url" {
 }
 
 variable "webhook_content_type" {
-  description = "Webhook Content Type (E.g. application/json)"
-  default     = "application/json"
+  description = "Webhook Content Type (E.g. json)"
+  default     = "json"
```

Comment:
Are you sure this is right? This doesn’t look like a valid content type.

Reply:
@osterman yes, I am sure this is right. See #4 `application/json`, is an "invalid type" in this case.